components/EmailValidator.vue

```diff
@@ -120,15 +126,24 @@ export default {
           }
         }
 
-        const validate = await this.$refs.email?.validate()
-        if (validate) {
-          this.$emit('update:valid', validate.valid)
+        const meta = this.$refs.form?.getMeta();
+        if (meta) {
+          this.$emit('update:valid', meta.valid)
         }
       },
     },
   },
   methods: {
-    validateEmail(value) {
+    async checkValidDomain(value) {
+      const domain = value.substring(value.indexOf('@') + 1)
+      const url = new URL('https://dns.google/resolve');
+      url.search = new URLSearchParams({ name: domain }).toString();
+      const googleResponse = await fetch(url).then(response => response);
```

Comment:
Probably worth a `try...catch`.  If that call fails for any reason, potentially including privacy blockers, then we'd want to ignore that failure.

Reply:
Yeah, good point. I'll add wrapper